packages/gatsby-plugin-offline/README.md

```diff
@@ -166,6 +166,28 @@ const options = {
 }
 ```
 
+You can also use [Workbox CLI](https://developers.google.com/web/tools/workbox/modules/workbox-cli) to generate this file automatically. 
+
+```
+workbox wizard
+? What is the root of your web app (i.e. which directory do you deploy)? public/
+? Which file types would you like to precache?
+? Where would you like your service worker file to be saved? public/sw.js
+? Where would you like to save these configuration options? workbox-config.js
+```
+
+Copy the content from `workbox-config.js` into your `gatsby-plugin-offline` options.
+
+For instance, the following configuration does the trick for a full offline website
+
+```
+  "globDirectory": "public/",
+  "globPatterns": [
+    "**/*.{html,js,json,png,jpg,webmanifest,woff,woff2,ttf,eot,css,mjs}"
```

Comment:
These globPatterns are different than those generated following the instructions above. Is this intentional?